uaclient/cli.py

```diff
@@ -77,7 +78,7 @@ def attach_parser(parser=None):
     parser.add_argument(
         'token', nargs='?',
         help=('Optional token obtained from Ubuntu Advantage dashboard: %s' %
```

Comment:
Should we still be calling it the "dashboard" in this message?

Reply:
changed, also fixed the help docs on --otp